README.md

```diff
@@ -99,149 +144,169 @@ This Dockerfile currently contains all installed C libraries and the Python wrap
 ```c
-double ccl_linear_matter_power(ccl_cosmology * cosmo, double k, double a);
-double ccl_nonlin_matter_power(ccl_cosmology * cosmo, double k, double a);
+double ccl_linear_matter_power(ccl_cosmology * cosmo, double k, double a,int * status);
+double ccl_nonlin_matter_power(ccl_cosmology * cosmo, double k, double a,int * status);
 ```
-Sigma_8 can be calculated by function **`ccl_sigma8`**, or more generally by function **`ccl_sigmaR`**, which computes the variance of the density field smoothed by spherical top-hat window function on a comoving distance **`R`** (in Mpc).
+Sigma_8 can be calculated by function **`ccl_sigma8`**, or more generally by function **`ccl_sigmaR`**, which computes the variance of the density field smoothed by spherical top-hat window function on a comoving distance `R` (in Mpc).
 ```c
-double ccl_sigmaR(ccl_cosmology *cosmo, double R);
-double ccl_sigma8(ccl_cosmology *cosmo);
-
-````
-These and other functions for different matter power spectra can be found in file *include/ccl_power.h*.
+double ccl_sigmaR(ccl_cosmology *cosmo, double R, int * status);
+double ccl_sigma8(ccl_cosmology *cosmo, int * status);
+```
+These and other functions for different matter power spectra can be found in file ***include/ccl_power.h***.
 
 ### Angular power spectra
-CCL can compute angular power spectra for two tracer types: galaxy number counts and galaxy weak lensing. Tracer parameters are defined in structure **CCL_ClTracer**. In general, you can create this object with function **ccl_cl_tracer_new**
+`CCL` can compute angular power spectra for two tracer types: galaxy number counts and galaxy weak lensing. Tracer parameters are defined in structure **`CCL_ClTracer`**. In general, you can create this object with function **`ccl_cl_tracer_new`**
 ````c
 CCL_ClTracer *ccl_cl_tracer_new(ccl_cosmology *cosmo,int tracer_type,
 				int has_rsd,int has_magnification,int has_intrinsic_alignment,
 				int nz_n,double *z_n,double *n,
 				int nz_b,double *z_b,double *b,
 				int nz_s,double *z_s,double *s,
 				int nz_ba,double *z_ba,double *ba,
-				int nz_rf,double *z_rf,double *rf);
+				int nz_rf,double *z_rf,double *rf, int * status);
 ````
-Exact definition of these parameters are described in file *include/ccl_cls.h*. Usually you can use simplified versions of this function, namely **ccl_cl_tracer_number_counts_new, ccl_cl_tracer_number_counts_simple_new, ccl_cl_tracer_lensing_new** or **ccl_cl_tracer_lensing_simple_new**. Two most simplified versions (one for number counts and one for shear) take parameters:
+Exact definition of these parameters are described in file ***include/ccl_cls.h***. Usually you can use simplified versions of this function, namely **`ccl_cl_tracer_number_counts_new`, `ccl_cl_tracer_number_counts_simple_new`, `ccl_cl_tracer_lensing_new`** or **`ccl_cl_tracer_lensing_simple_new`**. Two most simplified versions (one for number counts and one for shear) take parameters:
```

Comment:
More of a comment for the code than for the README: why do we have "new" at the name of each function name? I think we should rename these.

Reply:
I think that use of **`new`** should be a hint for the user to clean up after he is finished with some sort of **`free`**

Reply:
Ok, I'll think whether we want to change this eventually